README.md

```diff
@@ -72,6 +72,25 @@ or
 
 You need to download the [dataset](https://zenodo.org/record/7012043#.Y9jQV3bMKPo) and place it into the folder `./data/raw` such that the models are in `./data/raw/sap_sam_2022/models`.
 
+We provide an `autosetup.sh` file for automatic virtual environment setup with `venv`.
```

Comment:
The script will only work on Mac, right? This is worth a comment. Another idea would be to ensure it works well enough in other environments that can run bash scripts, I think the only real issue is homebrew atm...

Reply:
I am planning to go school this weekend because I need to finish a project - can test the script on a Linux machine there and will get back to you.
After making sure it works, I can add installing advice for Linux users. I assume this is what you meant with the homebrew issue?

Reply:
Exactly, so that we can assume, at least more or less, that it should work on Linux, as well as on Windows with a bash emulator.